### PR TITLE
Add selinux-policy packages into task image

### PR DESCRIPTION
**Pull request by marusak**
As needed for https://github.com/cockpit-project/cockpit/pull/15707

### CODE REVIEW

File: tasks/Dockerfile [hunk 1]
```diff
@@ -40,6 +40,8 @@ RUN dnf -y update && \
         rpmdevtools \
         rsync \
         sassc \
+        selinux-policy \
+        selinux-policy-devel \
```
Comment (martinpitt) [inline tasks/Dockerfile:44]: Dou you mind moving that after the `builddep` with a HACK comment pointing to the cockpit PR? We can drop this once it lands in fedora, as then builddep will install it  for us.

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: tasks/Dockerfile (hunk 1 above)
Comment (marusak) [reply tasks/Dockerfile:44]: Right, good call. Will do.

**Review by martinpitt — APPROVED** [review]
Cheers! Want to deploy this yourself, or want me to?

**Comment by marusak** [comment]
nvm, this needs to go into image.
I removed it from quay.io and retaged the old image.

**Pull request closed without merging**